tests/js/utils.js

```diff
@@ -250,8 +250,7 @@ export const provideModules = ( registry, extraData = [] ) => {
 		} )
 		.concat(
 			extraData.filter( ( { slug } ) => ! moduleSlugs.includes( slug ) ),
-		)
-    ;
+		);
```

Comment:
Thanks 😄

Reply:
Haha - I think it's my IDE being OCD actually. I had to revert a lot of changes it made because it thought they were prettier. This one must've slipped through the net...

Reply:
And thanks for finding those other instances!